django-stubs/contrib/gis/geos/geometry.pyi

```diff
@@ -10,15 +10,14 @@ class GEOSGeometryBase(GEOSBase):
     ptr_type: Any = ...
     destructor: Any = ...
     has_cs: bool = ...
-    __class__: Any = ...
     def __init__(self, ptr: Any, cls: Any) -> None: ...
     def __copy__(self) -> Any: ...
```

Comment:
`__copy__` and `__deepcopy__` can return `Self` type.

Reply:
You mean `from typing_extensions import Self`?

Reply:
Ah nope, got to use the `self: _T` trick still.